packages/templating-tools/package.js

```diff
@@ -8,15 +8,7 @@ Package.describe({
 Package.onUse(function(api) {
   api.use([
     'underscore@1.0.9',
-    'ecmascript@0.5.8',
-
-    // minifier-js is a weak dependency of spacebars-compiler; adding it here
-    // ensures that the output is minified.  (Having it as a weak dependency means
-    // that we don't ship uglify etc with built apps just because
-    // boilerplate-generator uses spacebars-compiler.)
-    // XXX maybe uglify should be applied by this plugin instead of via magic
-    // weak dependency.
-    'minifier-js@1.2.14'
```

Comment:
So I think we still want minimization to happen and that can be a normal dependency on a new minifier.